main.cpp

```diff
@@ -1,475 +1,581 @@
+    //annual profit (double)
+    double annualProfit = 45362.78;
+    //price of coffee (float)
+    float priceCoffee = 3.40f;
+
+    struct Employee
+    {
+        std::string firstName = "Magnus";
+        std::string lastName = "Magnusson";
+        int employeeID = 12345;
+        bool fullyTrained = true;
+        float salary = 30000.00f;
+
+        void makeDeliveries();
+        void takeHoliday();
+        bool deservesPromotion(); // returns whether the employee is derving of a promotion
+
+
+
+    };
+
+    //host local artists work on walls
+    float getArtExhibitCommision(float percenatgeCommisionAgreed, float totalArtSales, CoffeeShop coffeeshop); //returns the total value of commission earn from the sale of all the art at this instance of CoffeeShop 
+    //charge customer
+    float getCustomerSale(float priceCoffee, int numberOfCoffees, int customerNumber); //returns the total value of the sale of all good for this customer
+    //train staff
+    void staffTraining(Employee employee, std::string trainingRequired);
+};
+
 /*
-Part 1d assignment:
-1) Fill in #5 - #9 below with plain-english UDTs for the 5 
-properties you created for UDT #10. 
-    example: 
-        If #10's first property was 'Engine', then `Thing 5)` 
-        will be `Engine`. 
-        You will need to provide 5 properties and 3 member 
-        functions of that Engine object in plain English.
-        Remember to pick properties that can be represented 
-        with 'int float double bool char std::string'.
-
-2) write the name of the primitive type you'll be using after 
-each property for UDTs 5 - 9.
-    You already did this for UDTs 1-4 in Part 1c.
-    Pick properties that can be represented with 
-    'int float double bool char std::string'
-    example: 
-        Display:
-            Number of Pixels (int)
-            Amount of Power consumed (milliwatt-hours) (float)
-            Brightness (double)
-            width in cm (int)
-            height in cm (int)
+Thing 2) School
+5 properties:
+    1) number of classrooms (int)
+    2) position in leauge table (int)
+    3) number of teachers (int)
+    4) number of pupils (int)
+    5) student average grade (float)
+3 things it can do:
+    1) train teachers
+    2) students christmas play
+    3) close for holidays
 */
 
+struct School
+{
+    //number of classrooms (int)
+    int numClassrooms = 111;
+    //position in leauge table (int)
+    int leagueTablePosition = 13;
+    ///number of teachers (int)
+    int numTeachers = 54;
+    //number of pupils (int)
+    int numPupils = 1111;
+    //student average grade (float)
+    float aveGrade = 3.45f;
+
+    //train teachers
+    void teacherTraining(bool internalTraining, std::string trainingRequired);
+    //students christmas play
+    void christmasPlay(int numberOfStudentsInCast, std::string nameOfPlay, int rehearsalDays);
+    //close for holidays
+    void holidayClosure(int howManyDaysClosed, std::string nameOfHoliday);
```

Comment:
Your function names do not match the plain-english descriptions.
They should match almost 1:1
- [ ] revise